integration_tests/test_commands/tests/test_kbenv.py

```diff
@@ -0,0 +1,76 @@
+"""Test running kbenv commands."""
+import os
+import tempfile
+from subprocess import check_output
+from integration_tests.test_commands.test_commands import Commands
+
+
+
+class TestKBEnv(Commands):
+    """Tests kbenv subcommand."""
+
+    TEST_NAME = __name__
+
+    tmp_stack = None
+
+    stack_path = None
+
+    env = None
+
+    def init(self):
+        """Initialize eks."""
+        pass  # pylint: disable=unnecessary-pass
+
+    def run(self):
+        """Run tests."""
+
+        # setup context
+        tmp_root = tempfile.gettempdir()
+        self.tmp_stack = tempfile.TemporaryDirectory(dir=tmp_root)
+        self.stack_path = self.tmp_stack.name + '/k8s-tf-infrastructure'
+        self.env = dict(
+            os.environ,
+            CI='1',
+            DEPLOY_ENVIRONMENT='dev',
+            KUBECONFIG='{0}/.kubeconfig'.format(self.stack_path))
+
+        # create project from k8s-tf-repo sample
+        check_output(
```

Comment:
It *might* be worth considering swapping these `check_output(...).decode()` calls to `check_call(...)` -- since we're not actually doing anything with the returned stdout, it _feels_ slightly unnecessary to turn the output into strings that aren't assigned to anything.

On the flip side, there's almost certainly no performance change that can be measured from this and perhaps there's more value in keeping these consistent with the statements where we do care about the output.